portal/templates/redesign/teach_new/base_registering.html

```diff
@@ -14,4 +16,56 @@
 {% endblock rightMenuButtons %}
 
 {% block content %}
+<div class="banner banner--teacher--plain">
+    <h1 class="col-sm-offset-2">Welcome, {{ user|make_into_username }}</h1>
+    <h4 class="col-sm-offset-2">Everything you need to start coding with your class is here. Let&rsquo;s set you up.</h4>
+    <div class="arrows">
+        {% block setupCompleteStep %}
+        <div class="onboarding__step onboarding__step--todo">
+            <div class="onboarding__step__text">
+                <img title="To do" alt="Step to do" src='{% static "portal/img_new/step_undone.png" %}'>
```

Comment:
Avoid inline images if you can. There are a few times they are a good idea, but this project doesn't have many of them!